Gruntfile.js

```diff
@@ -81,15 +81,15 @@ module.exports = function ( grunt ) {
 	grunt.loadNpmTasks( 'grunt-exec' );
 
 	grunt.registerTask(
-		'teststyle',
+		'lint',
 		'Tests files for code style and code quality.',
 		[ 'eslint' ]
 	);
 
 	grunt.registerTask(
 		'test',
 		'Runs unit tests as well as checks code style/quality.',
-		[ 'teststyle', 'exec:jest' ]
```

Comment:
lint should be added to the build task down below I think, now that it's no longer covered by the test task which build includes.